test/html-to-react-tests.js

```diff
@@ -276,6 +276,15 @@ describe('Html2React', function () {
       assert.strictEqual(reactElem.props.onClick, 'alert(\'hello!\')');
     });
 
+    it('should return a valid HTML string with onclick attributes', function () {
```

Comment:
I realize now that the `onclick` attribute is tested at line 271, which verifies that you get a React component with a valid `onClick` property. As such, isn't your issue actually in React itself, since what happens is that React doesn't render the `onclick` HTML property?

As such, I think this PR might be dead in the water.